GitInfo.txt

```diff
@@ -1 +1 @@
-6.0.100-preview.7
+6.0.100-rc.1
```

Comment:
If this is going into p7, some of the commits should be removed, no?

Reply:
This is currently targeted at main.